modules/ROOT/pages/6.8-release-notes.adoc

```diff
@@ -303,20 +303,30 @@ In {productname} 6.8, the visible text label _(or icon name where icons are pres
 
 {productname} 6.8 also includes the following additions:
 
-////
-waiting for confirmation on these new feature tickets.
-=== Detect markdown on paste, and convert to HTML
-// #TINY-10199
+=== New `sandbox_iframes` option that controls whether iframe elements will be added a `sandbox=""` attribute to mitigate malicious intent.
+// #TINY-10348
+Recently it was discovered that iframes could embed resources that can cause nefarious effects when inserted into the editor via scripting and other mechanisms. This was especially problematic if the iframe had scripts that triggered automatic downloads of malicious files, or if the iframe resource is hosted on the same origin as the editor, which could lead to highly impactful scripting in the same context. In addition, since iframes can be modified to have very small, near-invisible dimensions, they could be unintentionally copied and pasted by users into the editor.
 
-=== Wrap markdown lib with API calls
-// #TINY-10198
+{productname} 6.8 addresses this, by introducing a new `sandbox_iframes` boolean option. When set to `true`, it will add a `sandbox=""` attribute to all iframes inserted into the editor. According to MDN docs, this will restrict the iframe's embedded resource from performing potentially malicious actions including scripting, file downloads, browser popups, passing the same-origin policy, among others.
```

Comment:
Add this link to "MDN docs" - https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#sandbox

Reply:
```suggestion
{productname} 6.8 addresses this, by introducing a new `sandbox_iframes` boolean option. When set to `true`, it will add a `sandbox=""` attribute to all iframes inserted into the editor. According to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#sandbox[MDN docs], this will restrict the iframe's embedded resource from performing potentially malicious actions including scripting, file downloads, browser popups, passing the same-origin policy, among others.
```